sympy/core/tests/test_relational.py

```diff
@@ -525,6 +569,9 @@ def test_inequalities_symbol_name_same():
     A = (x, y, S.Zero, S.One/3, pi, oo, -oo)
     for a in A:
         for b in A:
+            # do not check undefined comparisons of infinite values
+            if a in (-oo, oo) and a == b:
+                continue
```

Comment:
Is this still needed?

Reply:
Nope. Test still passes when I remove that

Reply:
But it's not worth pushing and resetting the tests for just that

Reply:
I will push that with a couple of other small improvements